Descension/Assets/Scripts/Items/Pickups/HealthItem.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using Actor.AI;
+using Actor.Player;
+using Managers;
+using UnityEngine;
+using Util.Enums;
+using Util.Helpers;
+
+namespace Items.Pickups
+{
+    public class HealthItem : EquippableItem
+    {
+        public static String Name = "Health Potion";
+        
+
+        public override string GetName()
+        {
+            return Name;
+        }
+
+        // override just creates class instance, passes in editor set values
+        public override Equippable CreateInstance(int slotIndex, int quantity)
+        {
+            return new HealthPotion(slotIndex, quantity, maxQuantity, inventorySprite);
```

Comment:
Should add healAmount variable to HealthItem and to the HealthPotion  constructor and pass it in through the HealthItem class, that way we can adjust it in editor.